controllers/agents.js

```diff
@@ -942,9 +942,9 @@ router.delete('/groups/:group_id', function(req, res) {
 router.delete('/', function(req, res) {
     logger.debug(req.connection.remoteAddress + " DELETE /agents");
 
-    var data_request = { 'function': 'DELETE/agents/', 'arguments': {} };
-    var filter_body = { 'ids': 'array_numbers', 'purge': 'boolean' };
```

Comment:
Should the `purge` argument be removed from the `body`? If it is sent in a JSON, the request will fail. I think it could be much better to accept the argument in the body and in the query.